static/src/javascripts/projects/common/modules/ui/message.js

```diff
@@ -50,10 +52,12 @@ class Message {
         this.customJs = opts.customJs || noop;
         this.customOpts = opts.customOpts || {};
         this.$footerMessage = $('.js-footer-message');
+        this.$siteMessage = $('.js-site-message');
     }
 
     show(message: string): boolean {
         const siteMessage = $('.js-site-message');
```

Comment:
Now you've defined `this.$siteMessage` do we need to declare `siteMessage` here and query the dom again when doing so? (I'm assuming the results of `$('.js-site-message')` are the same when we call it in `constructor` and `show`).

Reply:
Yeah, it's not a big issue, but as you made the assumption in `trapFocus()` I figured it'd be safe to do the same in `show()` and just use `this.$siteMessage` there too. 

I'm not a fan of exposing properties by adding them to `this`, but it seem to be the done thing in this module when functions need to share variables as it's been setup as a `Class` (something I'd like to look at changing).